### DISCUSSION TITLE
Project's Elevator Pitch

### DISCUSSION THREAD

**Discussion by ebouchut** [bet-no-loss/bet-no-loss#25] [Ideas]
@Tanteli What is your take on this elevator pitch aimed to describe how the project should work?

> Players bet on a previously registered sport event during week 1.  
> The total amount of bets (all players included) is then staked in DeFi during week 2.  
> Finally at the end of week 3, among all potential winners only 3 are drawn by lot to have the opportunity to withdraw the interests accrued in DeFi minus the fees, each one being proportional to what they initially staked.

**Comment by Tanteli** [comment]
Sounds good. However all winners would be allowed to withdraw the interests, instead of 3 as mentionned.

**Comment by ebouchut** [comment]
👍 available in 2b43a5423825e27e0d8e296ae6cd8dd19cb7f678
